rest-client/mhs/inbound/Solicited-request.http

```diff
@@ -0,0 +1,13252 @@
+# In inbound main.py, change ssl_ctx.verify_mode = ssl.CERT_REQUIRED to ssl.CERT_NONE or ssl.CERT_OPTIONAL
+# This request will be throw 500 server error because of RefToMessageId. This is the refers to message id
+# to the 'original' message from outbound. Add this id to state database and specific workflow. 
+
+# @name Solicited-request
+POST {{HOSTNAME}}:{{INBOUND-PORT}}/ HTTP/1.1
+Host: rmmid.nis1.national.ncrs.nhs.uk
+SOAPAction: urn:nhs:names:services:gp2gp/COPC_IN000001UK01
+Content-Length: 1028570
```

Comment:
No! Please do not commit such large files!
This is an example request that can be much smaller!
Look at the attachments. The b64 object i soooo large. As an example you can use something much smaller like "Hello, World!" encoded

Reply:
@bartek-sarul , @Broom94 ,  I've just replaced it with "Hello World"